Add header for optimizing bg energy usage

cc-cortes opened the pull request:
Point to the Optimize background activity page

cc-cortes committed:
Add header for optimizing bg energy usage

Point to the Optimize background activity page

msftclas commented:
@cc-cortes,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

PatrickFarley commented:
The optimize-background-activity doc doesn't live in the launch-resume section, so we typically wouldn't place it in the index there. I will ask if an exception should be made, since it pertains to both categories.

PatrickFarley committed:
relocate link

I've moved this to the Background tasks section, as a "See also" link. I didn't put it in the table because that is reserved for the topics that live in this section of docs.
